scripts/ssl/EXAMPLE-CERT-CREATION.md

```diff
@@ -4,7 +4,7 @@
 
 Please execute the following script `bash create-tls-certs.sh` to create all necessary certificates for the complete setup of all related components.
 
-## Adoption
+## Adaptation
 
 Please adjust the configuration files inside the [config](./config) folder and adopt the corresponding values for the req_distinguished_names and subjectAltNames based on your organisation and configuration. 
```

Comment:
there is another `adopt` here `...folder and adopt the corresponding values...` :-/

Reply:
Thank you. Done